test/e2e/tls_test.go

```diff
@@ -53,7 +53,11 @@ func TestAntreaApiserverTLSConfig(t *testing.T) {
 
 	controllerPod, err := data.getAntreaController()
 	assert.NoError(t, err, "failed to get Antrea Controller Pod")
-	controllerPodName := controllerPod.Name
+	controllerPodNode := controllerPod.Spec.NodeName
+	node := getNodeByName(controllerPodNode)
+	assert.NotNil(t, node, "failed to get the Node")
+	nodeIPv4 := node.ipv4Addr
+	nodeIPv6 := node.ipv6Addr
```

Comment:
Could Ensure that the retrieved addresses are not empty
with assert.NotEmpty

Reply:
At this point one of the addresses can be empty (single-stack cluster), which is fine